adding_metadata_sql.md

```diff
@@ -0,0 +1,181 @@
+
+### Step 1: Add new experiments
+
+```sql
+insert into experiments (name, start_date, end_date, user_id) 
+values ('MAC Season 6: Sorghum BAP', '2018-04-06', '2018-08-01', 'some text', 'some text', 6000000004)
+```
+
+### Step 2: Add new sites
+
+_must provide shapefile for a site to get geometry_
+
+```sql
+insert into sites (city, state, country, sitename) 
+values ('Maricopa', 'Arizona', 'USA', 'MAC Field Scanner Season 7 Range 9 Column 15')
+```
+
+For the TERRA REF Project, plot definitions may be copied from previous season if same plots are used.
+
+```sql
+with season6 as (
+  select city, state, replace(sitename, 'Season 4', 'Season 6') as sitename, greenhouse, geometry, time_zone from sites where sitename like '%Season 4%' 
+)
+insert into sites (city, state, sitename, greenhouse, geometry, time_zone) select * from season6
+```
+
+
+### Step 3: Add new treatments
+
+```sql
+insert into treatments (name, definition, control) 
+values ('MAC Season 6: Sorghum', 'some text', 't')
+```
+
+### Step 4: Add new cultivars
+
+_each cultivar should be associated with a specie_
+
+_new species must be added to species table before referencing_
+
+```sql
+insert into cultivars (name, specie_id) 
+values ('RIL-CS27_(TX2910/(Macia/R07007)-CS44)-CSF1-PRF2-CS27', 2588)
+```
+
+### Step 5: Add new citations
+
+```sql
+insert into citations (author, year, title) 
+values ('Newcomb, Maria', 2016, 'Maricopa Agricultural Center Field Activities')
+```
+
+### Step 6: Associate experiments with sites
+
+```sql
+insert into experiments_sites (experiment_id, site_id) values ((select id from experiments where name = 'MAC Season 6: Sorghum BAP'),
+(select id from sites where sitename = 'MAC Field Scanner Season 6 Range 1 Column 1 E'))
+```
+
+When adding a new season for the TERRA REF project, a statement like the following can be used for associating experiments and sites since MAC Field Center sites are consistently named following the format `MAC Field Scanner Season x Range a Column b`.
+
+```sql
+insert into experiments_sites (experiment_id, site_id) 
+   select e.experiment_id, s.site_id 
+        from (select id as experiment_id from experiments where name = 'MAC Season 6: Sorghum BAP')  as e 
+     cross join 
+         (select id as site_id from sites where sitename like 'MAC Field Scanner Season 6%') as s
+```
+
+## Step 7: Associate experiments with treatments
+
+```sql
+insert into experiments_treatments (experiment_id, treatment_id) 
+values ((select id from experiments where name = 'MAC Season 6: Sorghum BAP'),
+(select id from treatments where name = 'MAC Season 6: Sorghum'))
+```
+When adding a new season for the TERRA REF project, a statement like the following can be used to associate experiments with treatments since experiment and treatment names are usually named following the format `MAC Season x: subexperiment name`.
```

Comment:
Again, this could be clearer.  At a minimum, you could say "... can be used to associate all season 6 experiments with all season 6 treatments ..." and emend the format to `MAC Season x: [sub-experiment or treatment name]` or just `MAC Season x: ...`.